src/redux/store.js

```diff
@@ -0,0 +1,10 @@
+import { configureStore } from '@reduxjs/toolkit';
+import booksReducer from './books/booksSlice';
+
+const store = configureStore({
+  reducer: {
+    books: booksReducer,
+  },
```

Comment:
- [ ] Great job working on the functionality of the reducers and actions so far but I think it would be great if you add in the store the reducer for the categories as well beside the books reducer so kindly import it and add it in the store 🔥✨🚀💯🙏🏻

## You should include in the store as well 💯🙏🏻🚀✨
![image](https://github.com/ybadr99/bookstore/assets/58553711/fa8a13aa-0655-4669-bad6-8745d6e50303)